docs/site/DEVELOPING.md

```diff
@@ -113,6 +114,30 @@ npm script `lint:fix`.
 ```sh
 npm run lint:fix
 ```
+## File naming convention
+
+For consistency, we follow
+[Angular's file naming convention](https://angular.io/guide/styleguide#separate-file-names-with-dots-and-dashes).
+It also helps to derive the usage of files by inspecting the names.
```

Comment:
I don't think we need `also` since there is no prior point made in this paragraph.